scripts/build.js

```diff
@@ -116,12 +118,21 @@ function recursiveMerge(schema) {
   return schema;
 }
 
-// Merge instances of `allOf` in methods.
+// Set additionalProperties: false for all objects
+function setAdditionalProperties(schema) {
```

Comment:
I would rather each method explicitly be marked `false` for `additionalProperties`. This is so that we keep data necessary to understanding the schema as close to the actual schema definition as possible so viewers can from a cursory glance understand why they may be failing the schema.

Reply:
Sounds good. I've made the required changes but there will be conflicts with #437 so waiting until that's merged to update this PR.